src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -1987,6 +1987,9 @@ public static class UI
          public static readonly string MolWeight = "Molecular weight [g/mol]";
          public static readonly string CalculateBioavailability = "Calculate Bioavailability";
          public static readonly string CalculateDDIRatio = "Calculate DDI Ratio";
+         public static readonly string CalculateAll = "Calculate once";
+         public static readonly string RunForResults = "Include Venous Blood to the outputs and run for results";
```

Comment:
@abdelr This is the service here
SimulationPersistableUpdater
It has an overload for POP Simulation. 
```
 public void UpdatePersistableFromSettings(IndividualSimulation individualSimulation)
      {
         UpdateSimulationPersistable(individualSimulation);

         var organism = individualSimulation.Model.Root.Container(Constants.ORGANISM);

         individualSimulation.Compounds.Each(compound => addRequiredOutputForSimulation(organism, compound));
      }

      public void UpdatePersistableFromSettings(PopulationSimulation populationSimulation)
      {
         UpdateSimulationPersistable(populationSimulation);
      }
```

WE just nee dto make sure the two outputs are doing the same stuff. Because sometime, it's not venous blood that you need but Perivpheral venous blood etc.. So I would simply unify the call to be the same for all simulations and that's it

A few more entries calculated, but this is required to calculate PK Analysis. And if this should be done on demand, I would add ALL the one required for PK calculation and not only Venous  Blood because this is wrong.